From khali@linux-fr.org Tue Apr 25 05:22:12 2006
Date: Tue, 25 Apr 2006 14:22:14 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: HWMON: lm83: Documentation update
Message-Id: <20060425142214.1ff4b7b2.khali@linux-fr.org>
Content-Disposition: inline; filename=hwmon-lm83-documentation-update.patch

One more motherboard confirmed to have an LM83 temperature sensor chip.
Thanks to Steven Hardy for reporting.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/hwmon/lm83 |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/Documentation/hwmon/lm83
+++ gregkh-2.6/Documentation/hwmon/lm83
@@ -35,6 +35,7 @@ contact us. Note that the LM90 can easil
 
 Confirmed motherboards:
     SBS         P014
+    SBS         PSL09
 
 Unconfirmed motherboards:
     Gigabyte    GA-8IK1100
